pyproject.toml

```diff
@@ -39,10 +40,11 @@ pytest-cov = "^2.8"
 mypy = "^0.730"
 flake8-bugbear = "^19.8"
 pytest-django = {version = "^3.5", optional = true}
-pytest-mypy-plugins = "^1.0"
+pytest-mypy-plugins = "~1.0"
```

Comment:
is there any reason for this change?

Reply:
Yes, see [this comment](https://github.com/strawberry-graphql/strawberry/pull/191#issuecomment-546462128).

Reply:
I merged master into this branch and there it has been bumped to `^1.1` and is working. :)